src/scripts/main.js

```diff
@@ -1 +1,68 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(reject, 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  document.addEventListener('contextmenu', () => {
+    resolve('Second promise was resolved');
+  });
+
+  if (reject) {
+
+  }
+});
+
+const thirdPromise = new Promise((resolve) => {
+  let leftClick = false;
+  let rightClick = false;
+
+  document.addEventListener('click', () => {
+    leftClick = true;
+
+    if (leftClick && rightClick) {
```

Comment:
move into helper function to not duplicate

Reply:
not fixed. you should function to check if both clicks were made. if you have questions, feel free to ask in the chat
![image](https://user-images.githubusercontent.com/98764641/226311447-c2ae4c52-945a-47f7-ab57-3571985326d2.png)